x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/security/authz/store/ReservedRolesStoreTests.java

```diff
@@ -681,6 +681,27 @@ public void testKibanaSystemRole() {
             assertThat(kibanaRole.indices().allowedIndicesMatcher(DeleteIndexAction.NAME).test(mockIndexAbstraction(index)), is(true));
         });
 
+        // read-only indices for Endpoint events (to build timelines)
+        Arrays.asList(".logs-endpoint.events.process-default-" + randomAlphaOfLength(randomIntBetween(0, 13))).forEach((index) -> {
+            assertThat(kibanaRole.indices().allowedIndicesMatcher("indices:foo").test(mockIndexAbstraction(index)), is(false));
+            assertThat(kibanaRole.indices().allowedIndicesMatcher("indices:bar").test(mockIndexAbstraction(index)), is(false));
+            assertThat(kibanaRole.indices().allowedIndicesMatcher(GetIndexAction.NAME).test(mockIndexAbstraction(index)), is(true));
+            assertThat(kibanaRole.indices().allowedIndicesMatcher(CreateIndexAction.NAME).test(mockIndexAbstraction(index)), is(false));
```

Comment:
I'm not sure we can assert this if we need the Kibana system user to be able to roll over the index as tested below. Historically, rollover internally called `CreateIndexAction` so the ability to create the next index was an inherent requirement to be able to use rollover. That's not the case anymore and it would be technically possible to distinguish the two today, but we haven't (yet) added a privilege that just allows rollover and disallows other index creation.

My recommendation would be to drop this line from the test and use `manage` as I describe in my other comment, but I'm less familiar with the contents of these indices and any particular requirements we have around them.

Reply:
Got it, thanks!